core/src/main/java/org/mule/registry/SimpleRegistry.java

```diff
@@ -77,13 +77,16 @@ protected void doRegisterObject(String key, Object object, Object metadata) thro
 
         doPut(key, object);
 
-        try
-        {
-            getLifecycleManager().applyCompletedPhases(object);
-        }
-        catch (MuleException e)
+        if (!(object instanceof MuleContext))
```

Comment:
if this registry has completed phases is because the muleContext is already started. Don't start it again.